worker/runfmusite/testcase.py

```diff
@@ -0,0 +1,233 @@
+# -*- coding: utf-8 -*-
+"""
```

Comment:
Also in this version we are missing the appropriate copyright to IBPSA.

Reply:
added.